src/h5web/visualizations/heatmap/config.ts

```diff
@@ -13,8 +13,8 @@ interface HeatmapConfig {
   colorMap: ColorMap;
   setColorMap: (colorMap: ColorMap) => void;
 
-  hasLogScale: boolean;
-  toggleLogScale: () => void;
+  scaleType: ScaleType;
```

Comment:
you may need to update the storage whitelist

Reply:
:fishing_pole_and_fish: